core/storage/user/gae_models.py

```diff
@@ -1170,9 +1173,12 @@ def export_data(user_id: str) -> Dict[str, Union[List[str], float, None]]:
             UserSettingsModel.get_multi(user_model.creator_ids)
         )
         filtered_creator_user_models = []
-        for creator_user_model in creator_user_models:
-            # Ruling out the possibility of None for mypy type checking.
-            assert creator_user_model is not None
+        for index, creator_user_model in enumerate(creator_user_models):
```

Comment:
I'm not sure you can block a data export because of this. Can you treat it in a similar way to "user_model is None" above -- i.e. just omit the problematic data?

Reply:
As per the discussion, now we are skipping the None case.

Reply:
As per the discussion, now we are skipping the None case.